src/util/mirageServer.js

```diff
@@ -35,6 +36,14 @@ export function makeServer() {
         }
       );
 
+      this.get(
+        'http://localhost:8000/api/v1/auth/showMe',
+        (schema, request) => {
+          let attrs = JSON.parse(request.requestBody);
+          console.log(attrs);
+        }
```

Comment:
This should return something so that when this mock is called it will return something as the `response.data`